packages/@aws-cdk/aws-backup/test/plan.test.ts

```diff
@@ -70,6 +70,70 @@ test('create a plan and add rules', () => {
   });
 });
 
+test('create a plan and add rules - add BackupPlan.AdvancedBackupSettings.BackupOptions', () => {
+  // GIVEN
+  const vault = new BackupVault(stack, 'Vault');
+  const otherVault = new BackupVault(stack, 'OtherVault');
+
+  // WHEN
+  const plan = new BackupPlan(stack, 'Plan', {
+    windowsVss: true,
+    backupVault: vault,
+    backupPlanRules: [
+      new BackupPlanRule({
+        completionWindow: Duration.hours(2),
+        startWindow: Duration.hours(1),
+        scheduleExpression: events.Schedule.cron({
+          day: '15',
+          hour: '3',
+          minute: '30',
+        }),
+        moveToColdStorageAfter: Duration.days(30),
+      }),
+    ],
+  });
+  plan.addRule(BackupPlanRule.monthly5Year(otherVault));
+
+  // THEN
+  expect(stack).toHaveResource('AWS::Backup::BackupPlan', {
```

Comment:
use `toHaveResourceLike` and remove all the properties under BackupPlan except AdvancedBackupSettings (no need to test entire object since we just care about that property)